docs/node/mainnet/upgrade-log.md

```diff
@@ -5,6 +5,204 @@ node operators' deployments.
 
 They are enumerated and explained in this document.
 
+## 2023-11-29 (8:30 UTC) - Eden Upgrade {#eden-upgrade}
+
+:::caution
+
+For a detailed view of the Eden Upgrade process, please refer to the
+[Eden Upgrade] page.
+
+:::
+
+[Eden Upgrade]: ./eden-upgrade.md
+
```

Comment:
This should probably also include a short section on the GLIBC-related changes?

Reply:
Eden upgrade page already has this short section. Added now also this section in the Upgrade log.